bitbots_hcm/package.xml

```diff
@@ -22,13 +22,14 @@
   <depend>geometry_msgs</depend>
   <depend>humanoid_league_msgs</depend>
   <depend>irobot_events_executor</depend>
-  <depend>pybind11</depend>
-  <depend>Python3</depend>
-  <depend>PythonLibs</depend>
+  <depend>pybind11-dev</depend>
   <depend>rclpy</depend>
   <depend>ros2_python_extension</depend>
   <depend>sensor_msgs</depend>
   <depend>std_msgs</depend>
+  <depend>python3-sklearn</depend>
```

Comment:
Are you sure that all of this works with the versions provided in jammy, because some versions that are shipped in Ubuntu are quite old.

Reply:
Yes, sklearn on Jammy is the exact version that was pinned in our requirements file. Actually, any newer version breals our code.